dev.yml

```diff
@@ -0,0 +1,19 @@
+name: sewing-kit
+type: node
+up:
+  - node:
+      version: v8.10.0
+      yarn: true
+  - custom:
+      name: lerna bootstrap
+      met?: 'false'
+      meet: yarnpkg run lerna bootstrap
+
+commands:
+  __default__: start
+  build: yarnpkg build
+  check: yarnpkg check
+  lint: yarnpkg lint
+  start: echo "Running quilt is not really a concept that makes sense."
```

Comment:
I think it's worth mentioning that quilt is not an application or service that starts.